### PR TITLE
Make publisher errors slient

### PR DESCRIPTION
**Pull request by jnhyperion**
- LogPlugin should be loaded in the first, because some error message could be printed in log plugin during other plugin loading
- Title in plugin manage setting page cannot show properly

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -45,6 +45,7 @@ and this project adheres to http://semver.org/spec/v2.0.0.html[Semantic Versioni
 
 === Fixed
 
+- Fixed incorrect tittle in manage plugin settings
```
Comment (HelioGuilherme66) [inline CHANGELOG.adoc:48]: tittle? :D

**Comment by jnhyperion** [comment]
#2317

**Comment by jnhyperion** [comment]
this PR will cause unit test print lots of error message, need to avoid

File: CHANGELOG.adoc (hunk 1 above)
Comment (jnhyperion) [reply CHANGELOG.adoc]: updated

**Comment by jnhyperion** [comment]
fixed error msg in unit

**Review by HelioGuilherme66 — APPROVED** [review]

**Pull request merged**